lib/livebook/hubs/team.ex

```diff
@@ -200,4 +202,17 @@ defimpl Livebook.Hubs.Provider, for: Livebook.Hubs.Team do
   def update_file_system(team, file_system), do: Teams.update_file_system(team, file_system)
 
   def delete_file_system(team, file_system), do: Teams.delete_file_system(team, file_system)
+
+  def get_deployment_groups(team) do
+    TeamClient.get_deployment_groups(team.id)
+  end
+
+  def create_deployment_group(team, deployment_group),
+    do: Teams.create_deployment_group(team, deployment_group)
+
+  def update_deployment_group(team, deployment_group),
+    do: Teams.update_deployment_group(team, deployment_group)
+
+  def delete_deployment_group(team, deployment_group),
+    do: Teams.delete_deployment_group(team, deployment_group)
```

Comment:
For clarify, this is the API we will keep, almost everything else above this we can remove. :)